apps/client/src/components/ServicesConfig.vue

```diff
@@ -203,16 +196,14 @@ const reload = () => {
   >
     <DsfrButton
       v-if="Object.values(updated).keys() && Object.values(updated).map(v => Object.keys(v)).flat().length"
-      type="buttonType"
       label="Enregister"
-      data-testid="save"
-      @click="save"
+      data-testid="saveBtn"
```

Comment:
Il faut mettre à jour les éventuels tests end to end / components en adéquation avec ce changement de nom.